src/Sarif/Sarif.csproj

```diff
@@ -42,14 +42,16 @@
     <!--
     Marking these items as content causes them to be included in the NuGet package.
     -->
-    <Content Include="Schemata\sarif-schema-$(SchemaVersionAsPublishedToSchemaStoreOrg).json" />
-    <Content Include="Schemata\sarif-external-property-file-schema-$(SchemaVersionAsPublishedToSchemaStoreOrg).json" />
 
     <!-- Link=false: Don't display this item in Solution Explorer. -->
     <!-- This item will be added to the NuGet package, however. -->
     <Content Include="..\ReleaseHistory.md" Link="false" />
   </ItemGroup>
 
+  <ItemGroup>
+    <JsonSchemaFile Remove="Schemata\sarif-schema-2.1.0-rtm.1.json" />
```

Comment:
>sarif-schema-2.1.0-rtm.1.json [](start = 37, length = 29)

please confirm - this should be renamed to "sarif-schema" and in this PR?

Reply:
@mikefan@microsoft.com @larrygolding@comcast.net 

---
In reply to: [285745583](https://github.com/microsoft/sarif-sdk/pull/1494#discussion_r285745583) [](ancestors = 285745583)

Reply:
+ 1 external file schema rename to "sarif-external-property-file-schema"

---
In reply to: [285745631](https://github.com/microsoft/sarif-sdk/pull/1494#discussion_r285745631) [](ancestors = 285745631,285745583)

Reply:
no this name is right.

---
In reply to: [285746049](https://github.com/microsoft/sarif-sdk/pull/1494#discussion_r285746049) [](ancestors = 285746049,285745631,285745583)